components/esp_wifi/Kconfig

```diff
@@ -495,6 +495,7 @@ menu "Wi-Fi"
         bool "Enable DPP support"
         default n
         select ESP_WIFI_MBEDTLS_CRYPTO
+        select ESP_WIFI_MBO_SUPPORT
```

Comment:
We are trying to reproduce this issue and fix it. these are two separate features and should have no dependency on each other.

Reply:
I know they are very unrelated, however this change is the only one that's let dpp consistently work near perfectly, compared to barely working unless the listen channel was set to that of the ap and phone(dont know why its an issue since the qr has the channel + the esp sends on the correct channel anyways).
Without this fix, `esp_wifi_action_tx_req` would in most cases I tested, fail by sending the 
`WIFI_EVENT_ACTION_TX_STATUS` event with a status=1, which in turn esp_dpp.c would send the `ESP_ERR_DPP_TX_FAILURE` event to the callback.
Without this fix, it would take making the environmental conditions near perfect for me to get a moderate success rate, but with this fix, even going out of my way I cant intentionally cause the `WIFI_EVENT_ACTION_TX_STATUS` related error.

Reply:
Fwiw on 4.4.4 I could not reproduce MBO having an effect.  What helped for me most was fixing the fact that retry doesn't work as it is supposed to in the example code (s_dpp_stop_listening is not reset on start_listen), then just spamming retry on my phone.  It is possible this MBO feature does something, but it's gonna take some tough work to figure out why.  What isn't up for debate tho is that DPP is highly unreliable for more than just me, and we definitely need to dig in and understand why.